### PR TITLE
ocamlPackages.tsort: 2.0.0 -> 2.1.0

### PR DESCRIPTION
**Pull request by StridBot**
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://opam.ocaml.org/packages/tsort/tsort.2.1.0.

meta.description for ocamlPackages.tsort is: "Easy to use and user-friendly topological sort"

meta.homepage for ocamlPackages.tsort is: "https://github.com/dmbaturin/ocaml-tsort"

meta.changelog for ocamlPackages.tsort is: ""

###### Updates performed
- Version update

###### To inspect upstream changes

- [Compare changes on GitHub](https://github.com/dmbaturin/ocaml-tsort/compare/2.0.0...2.1.0)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- Warning: a test defined in `passthru.tests` did not pass

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- found 2.1.0 with grep in /nix/store/ri2b64ad63ypdg3n8r60abmk66yglbjm-ocaml4.13.1-tsort-2.1.0
- found 2.1.0 in filename of file in /nix/store/ri2b64ad63ypdg3n8r60abmk66yglbjm-ocaml4.13.1-tsort-2.1.0

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
1 total rebuild path(s)

1 package rebuild(s)

0 x86_64-linux rebuild(s)
0 x86_64-darwin rebuild(s)


First fifty rebuilds by attrpath
ocamlPackages.tsort
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Build yourself:
```
nix-build -A ocamlPackages.tsort https://github.com/StridBot/nixpkgs/archive/cceb6fa314ccdea7e8a3e5c1134475662815b016.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/ri2b64ad63ypdg3n8r60abmk66yglbjm-ocaml4.13.1-tsort-2.1.0
ls -la /nix/store/ri2b64ad63ypdg3n8r60abmk66yglbjm-ocaml4.13.1-tsort-2.1.0/bin
```

---

</details>
<br/>

### Pre-merge build results

NixPkgs review skipped

---

###### Maintainer pings

cc @vbgl for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 165677` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>1 package failed to build and are new build failure:</summary>
  <ul>
<li>soupault: log was empty</li>
  </ul>
</details>

**Comment by ulrikstrid** [comment]
@SuperSandro2000 soupault seems to have been updated in the meantime and now works with the updated tsort

File: pkgs/development/ocaml-modules/tsort/default.nix [hunk 1]
```diff
@@ -3,12 +3,12 @@
 buildDunePackage rec {
   pname = "tsort";
   useDune2 = true;
```
Comment (vbgl) [inline pkgs/development/ocaml-modules/tsort/default.nix:5]:
```suggestion
```

**Review by vbgl — COMMENTED** [review]
Looks good otherwise.

File: pkgs/development/ocaml-modules/tsort/default.nix (hunk 1 above)
Comment (ulrikstrid) [reply pkgs/development/ocaml-modules/tsort/default.nix]: Fixed

**Pull request merged**